.kokoro/presubmit/downstream-build.sh

```diff
@@ -15,15 +15,17 @@
 
 set -eo pipefail
 
-## Get the directory of the build script
-scriptDir=$(realpath "$(dirname "${BASH_SOURCE[0]}")")
-## cd to the parent directory, i.e. the root of the git repo
-cd "${scriptDir}/../.."
-
 if [ -z "${MODULES_UNDER_TEST}" ]; then
   echo "MODULES_UNDER_TEST must be set to run downstream-build.sh"
   exit 1
 fi
+# Use default value for REPOS_UNDER_TEST if unset. If set to empty string, maintain empty string.
+REPOS_UNDER_TEST=${REPOS_UNDER_TEST-"java-storage"}
```

Comment:
Is it intentional to have `java-storage` here?

Reply:
Yes. It's the only current repository with GraalVM tests that don't require cloud project resources.

Reply:
Thanks! Can we add comments to explain it?